CHANGELOG.develop

```diff
@@ -423,6 +423,17 @@ Benner and Paweł Siudak):
 - Renamed =spacemacs/mplist-get= function to =spacemacs/mplist-get-values=,
   renamed =spacemacs/plist-get= function to =spacemacs/mplist-get-value=, and
   refactored the latter function (thanks to Benjamin Reynolds)
+- Allow spaces and empty values in environment variable values (thanks to
+  Sylvain Benner)
+- Add support for fetching of environment variables on Microsoft Windows
+  (only tested on Windows 10) (thanks to Sylvain Benner)
+- Add GPG_AGENT_INFO and SSH_AGENT_PID to ignored environment variables
+  (thanks to Sylvain Benner)
+- Fix: potential undefined error with transient hooks during buffer restore
+  (thanks to Sylvain Benner)
```

Comment:
Transient hooks are a new feature since the last release, so I don't think this entry is needed.